CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * **[Fix]** Add missing network request error logging.
 
+### AppCenterCrashes
+
+* **[Improvement]** Catch additional exceptions on macOS.
```

Comment:
Can we be more vocal here and describe previous and new behavior?

Reply:
We should probably mention swizzling so that existing customers can be aware of it in case they don't want it.